### ISSUE TITLE
$clone after function that returns a struct causes unsupported method

### ISSUE THREAD

**Issue by RexHell** [bridgedotnet/Bridge#577]
I have encountered this problem.`
When I call a method that returns a struct, there will be a '$clone()' in the generated code, and the browser blames:
> object doesn't support "$clone" property or method.

The code is as follows:
```C#
using System;
using Bridge.Html5;

public class App
{
    public struct Unit
    {
    }

    public static Unit SomeMethod(int j)
    {
        return new Unit();
    }

    [Ready]
    public static void Main()
    {
        var v = SomeMethod(1);
    }
}
```

```javascript
/* global Bridge */

"use strict";

Bridge.define('App', {
    statics: {
        config: {
            init: function () {
                Bridge.ready(this.main);
            }
        },
        someMethod: function (j) {
            return new App.Unit();
        },
        main: function () {
            var v = App.someMethod.$clone()(1);
        }
    }
});

Bridge.define('App.Unit');
```

**Comment by LeonidVeriga** [comment]
Hi @RexHell! Thank you for the complete test case code that is easy to run. The issue is reproduced on the latest code. We will fix it in 1.10 release.

**Comment by LeonidVeriga** [comment]
The issue has been fixed and merged into `release1.10` branch.
